tests/cli/test_dolt.py

```diff
@@ -146,6 +147,41 @@ def test_dolt_log(create_test_table):
     assert previous_commit.message == message_one
 
 
+def test_dolt_log_number(create_test_table):
```

Comment:
DRY Dolt `test_dolt_log_*` with helper?

Reply:
I agree there's a more principled way to probably do this, I might want to do something like adding a .sql file that can create an appropriate setup state for most CLI+SQL tests at some point.